src/dll/fuse/fuse.c

```diff
@@ -505,6 +507,11 @@ static int fsp_fuse_core_opt_proc(void *opt_data0, const char *arg, int key,
             [sizeof opt_data->VolumeParams.FileSystemName / sizeof(WCHAR) - 1] = L'\0';
         memcpy(opt_data->VolumeParams.FileSystemName, L"FUSE-", 5 * sizeof(WCHAR));
         return 0;
+    case 'v':
+        arg += sizeof "volname" - 1;
+        memcpy(&opt_data->VolumeLabel, arg, sizeof opt_data->VolumeLabel);
```

Comment:
`arg` is a `const char *` and `VolumeLabel` is a `WCHAR[]`. The proper way to handle this is to do `MultiByteToWideChar`.